### ISSUE TITLE
Improvement of contribution docs.

### ISSUE THREAD

**Issue by Mr-Sunglasses** [scrapinghub/dateparser#990]
**_Improvement for the docs which cover about contribution to the repository , should be more elaborated because it does not do much help for the new contributors_** . @Gallaecio , @gavishpoddar . What are your opinions?

**Comment by Gallaecio** [comment]
There is always room for improvement, but since we do cover what I think is the most important topic for new contributors, how to extend language data, I wonder what other aspects of contribution are you looking for in the documentation. Could you be more specific, i.e. give examples of things that you were hoping to find in the documentation for contributors that were missing?

**Comment by Mr-Sunglasses** [comment]
@Gallaecio The docs should contain  more information about sub packages which were under dateparser repository. How they work and if you want to do any changes other than adding extend language data , which sub packages should you care about.

**Comment by Gallaecio** [comment]
So you are thinking of a general “map” of the code base?

**Comment by Mr-Sunglasses** [comment]
Yeah Exactly , @Gallaecio Just need your guidance and I'll start working on it. It will help many new contributers.
